test/gen_rpc_test_helper.erl

```diff
@@ -60,6 +61,8 @@ start_slave(Driver) ->
     ok = set_driver_configuration(Driver, ?SLAVE),
     %% Start the application remotely
     {ok, _SlaveApps} = rpc:call(?SLAVE, application, ensure_all_started, [?APP]),
+    %% debug(),
```

Comment:
Not sure if this is worth keeping. It may be useful for the next poor soul that needs to debug this.

Reply:
If it helps poor souls, I sympathize with keeping it here as a tip then (I might be such future soul :see_no_evil: ).